internal/services/zero_trust_access_policy/testdata/accesspolicyauthmethodconfig.tf

```diff
@@ -14,7 +14,4 @@ resource "cloudflare_zero_trust_access_policy" "%[1]s" {
       auth_method = "hwk"
     }
   }]
-  approval_required              = "false"
```

Comment:
These bool fields were handled in this branch by [internal/services/zero_trust_access_policy/normalizations.go](https://github.com/cloudflare/terraform-provider-cloudflare/pull/5889/files#diff-89fa7ecb9d7bf3c86b705e3c840efcc125ccbb781b16690f9a164caf44a53edc)

We don't want users to be required to set the value to false. On import the value is set to null. If a user wants to explicitly set the value to false after importing, they will need to run an apply again, but that seems fine given the tradeoffs.